Pod/Classes/CZPickerView.m

```diff
@@ -308,7 +308,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             cell.accessoryType = UITableViewCellAccessoryCheckmark;
         }
     }
-    if ([self.dataSource respondsToSelector:@selector(czpickerView:attributedTitleForRow:)]) {
+    if([self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)] && [self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)]){
```

Comment:
Because I think that normally there are two situations:
1. only text
2. image + text
it rarely happens only image for each table cell.
In order to make sure that text and image are all set, I choose to make two checks.
What's your opinion?